internal/xds/contour.go

```diff
@@ -13,19 +13,7 @@
 
 package xds
 
-import (
-	"context"
-	"fmt"
-	"strconv"
-	"sync/atomic"
-
-	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
-	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
-	"github.com/golang/protobuf/ptypes/any"
-	"github.com/sirupsen/logrus"
-)
+import "github.com/golang/protobuf/proto"
```

Comment:
maybe rename this file to `resource.go` since that's all it contains now.

Reply:
Yup makes sense. 👍